build.gradle.kts

```diff
@@ -28,8 +27,10 @@ dependencies {
     testRuntimeOnly("org.junit.jupiter", "junit-jupiter-engine", "5.8.2")
 }
 
-tasks.withType<KotlinCompile> {
-    kotlinOptions.jvmTarget = "11"
+java {
```

Comment:
I assume `java` toolchain alone is enough as it is the default for `kotlin` then as well? So we don't need to specify the `kotlin` toolchain?

Reply:
Doing this now via the kotlin plugin.